### PR TITLE
Support 2nd gen. miniserver

### LINKED ISSUE #78: Unable to connect to miniserver v2
ERROR 32529 --- [ctReadThread-36] cz.smarteon.loxone.LoxoneWebSocket: Can't parse response: Cannot construct instance of `cz.smarteon.loxone.app.MiniserverType`, problem: Invalid miniserver type specification, value=2

### PR DESCRIPTION
**Pull request by jimirocks**
Also add fallback to UNKNOWN type.

Resolves #78

### CODE REVIEW

File: src/main/java/cz/smarteon/loxone/app/MiniserverType.java [hunk 1]
```diff
@@ -2,10 +2,27 @@
 
 import com.fasterxml.jackson.annotation.JsonCreator;
 
+/**
+ * The type of the Loxone miniserver
+ */
 public enum MiniserverType {
 
+    /**
+     * Regular full size miniserver 1st gen. produced until Dec 2019
```
Comment (xzavrel) [inline src/main/java/cz/smarteon/loxone/app/MiniserverType.java]: It's still produced.

File: src/main/java/cz/smarteon/loxone/app/MiniserverType.java [hunk 2]
```diff
@@ -2,10 +2,27 @@
...
     REGULAR(0),
-    GO(1);
+    /**
+     * Miniserver GO
+     */
+    GO(1),
+    /**
+     * Regular full size miniserver 2nd gen. produced since Dec 2019
+     */
+    REGULAR_2(2),
```
Comment (xzavrel) [inline src/main/java/cz/smarteon/loxone/app/MiniserverType.java]: I would prefere REGULAR_V2

File: src/main/java/cz/smarteon/loxone/app/MiniserverType.java (hunk 1 above)
Comment (jimirocks) [reply src/main/java/cz/smarteon/loxone/app/MiniserverType.java]: OK so let's remove the produced note...

File: src/main/java/cz/smarteon/loxone/app/MiniserverType.java (hunk 2 above)
Comment (jimirocks) [reply src/main/java/cz/smarteon/loxone/app/MiniserverType.java]: renamed

**Pull request merged**